foundation/foundation-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -84,6 +84,18 @@
             <version>${archetypeVersion}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
S: are we sure we need `foundation-mda`?  It's not really a maven plugin, but rather a dependency that is fed to fermenter.  I would have though that the normal reactor would pick it up for the cache without explicitly being set.  

If it is that the build cache isn't picking up dependencies listed within plugins, I might just note that so it's easier to know what it is here in the future.

Reply:
I believe it is indeed the latter case, because after adding these two dependencies, it resolved the inconsistent behavior with the archetype tests. I will add a comment to contextualize

Reply:
Will also make sure the additions are not part of the plugins comment block